build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "1.1.7-beta"
+  version = "1.1.7-beta-2"
```

Comment:
The async endpoints are still in beta - but should be ok to use this version as they are backwards compatible (i.e. new).